docs/components/movement-sensor/wheeled-odometry.md

```diff
@@ -0,0 +1,69 @@
+You can access these readings through the [movement sensor API](/components/movement-sensor/#api).
+
+After configuring a `wheeled-odometry` movement sensor, you can operate your base with Viam's built-in services like the [navigation service](/services/navigation/).
+
+## Set-up requirements
+
+To prepare your robot, attach [encoders](/components/encoder/) to each of the position-reporting motors on your base to measure their rotation.
+
+- Pick out motors that can report their own position, like the [`roboclaw`](/components/motor/roboclaw/) or [`gpio` motors](/components/motor/gpio/) with [encoders](/components/encoder/#configuration).
+You can access this property of a configured motor through the [motor API's `GetProperties()`](/components/motor/#getproperties).
+- Configure your rover as a [wheeled base component](/components/base/wheeled/).
+- Configure each of the position-reporting motors [as motor components](/components/motor/).
+- Configure an [encoder component](/components/encoder/#configuration) for each of the encoders you attached to the position-reporting motors.
+- Then, proceed to [configure](#configuration) a `wheeledodometry` movement sensor with the name of each of the encoder components.
+
+{{% alert title="Tip" color="tip" %}}
+
+The `roboclaw` motor does not require you to configure [encoder components](/components/encoder/#configuration) for use with the `wheeled-odometry` movement sensor.
+It reports its own position with a built-in encoded motor.
+
+{{% /alert %}}
+
+## Configuration
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Select **Raw JSON** mode.
+Copy and paste the following:
+
+```json {class="line-numbers linkable-line-numbers"}
+"name" : "<your-wheeledodometry-movement-sensor-name>",
+"type" : "movement_sensor",
+"model" : "wheeledodometry",
+"attributes" : {
+    "base" : "<your-base-name>",
+    "left_motors" : ["<your-base-left-motor-name-1>", "<your-base-left-motor-name-2>"],
+    "right_motors" : ["<your-base-right-motor-name-1", "your-base-right-motor-name-2>"],
+    "time-interval-msec": <number>
+}
+```
+
+Fill in and edit the attributes as applicable.
+
+## Attributes
+
+The following attributes are available for `wheeledodometry` movement sensors:
+
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `base` | string | **Required** | The `name` of the [base](/components/base/) to which the encoders making up this movement sensor are wired. |
+| `left_motors` | object | **Required** | A struct holding the name of each of the bases' left [position-reporting motors](/components/motor/gpio/). |
```

Comment:
[nit] I might be wrong, but it seems more like a list than a struct? whatever you think. same for right motors on next line

Reply:
No, you're right! Thank you! Updating